### PR TITLE
Deprecated [EDA] - CA Add Application related fields to Contact

### LINKED ISSUE #1229: [EDA] - CA Add Application related fields to Contact
**Note: FOR INTERNAL USE ONLY. For support questions, please use the EDA Chatter group in [The Power of Us Hub](https://powerofus.force.com).
For feature requests, please use the Ideas tab in [The Power of Us Hub](https://powerofus.force.com).**

* **Do you want to request a *feature* or report a *bug*?**
Feature

* **What is the current behavior?**
Former Names, Citizenship Status and Date Deceased fields are currently unavailable on the Contact object.

* **What is the expected behavior?**
Former Names, Citizenship Status and Date Deceased fields are enabled for the Contact object.

* **What is the motivation / use case for changing the behavior?**
These fields on the Contact object will help shape a more complete 360 view of the individual and are characteristics that stay with the Contact for the duration of their education journey.

* **Acceptance Criteria**

  - **Given**: As a Salesforce admin at my educational institution 
  - **When**: I want to see every aspect of a Contact, informing a full 360 view of said Contact
  - **Then**: So that I can see what their former first name (first/middle/last) was, perhaps prior to marriage, so I can match transcripts or historical data. 

  - **Given**: As a Salesforce admin at my educational institution 
  - **When**: I want to see the citizenship status of my Contacts
  - **Then**: So that I can report to SEVIS (homeland security) per federal law, as well as manage any VISA sponsorship for my faculty and staff. 

  - **Given**: As a Salesforce admin at my educational institution 
  - **When**: I want to know the deceased date of any Contacts 
  - **Then**: So that I can avoid embarrassing marketing or communication mistakes addressing a deceased person, or so I can understand upcoming memorial dates and events.

### PR DESCRIPTION
**Pull request by m-kinkade**
# Critical Changes

# Changes
We have added fields to on the Contact object to accommodate Former Names, Citizenship Status and Date Deceased.  We have also included the new Citizenship Status field on the HEDA Contact Layout.

# Issues Closed
Closes #1229 

# New Metadata
Contact.Former_First_Name__c
Contact.Former_Middle_Name__c
Contact.Former_Last_Name__c
Contact.Citizenship_Status__c
Contact.Date_Deceased__c

# Deleted Metadata

# Testing Notes
https://salesforce.quip.com/bRXRAu2xIDxt#aSGACAqHLWk

### CODE REVIEW

**Comment by xinnanlisf** [comment]
Will let Mr. Kosgi @skosgi1 do a thorough review but it looks like we are missing translation objects here. ;)

**Comment by skosgi1** [comment]
> Will let Mr. Kosgi @skosgi1 do a thorough review but it looks like we are missing translation objects here. ;)

thanks Xinnan! @m-kinkade lets add translations files, let me know if you wanna meet regarding this.

File: src/objectTranslations/Contact-ca.objectTranslation [hunk 1]
```diff
@@ -3471,22 +3515,6 @@
         <help>Quin número de telèfon s&apos;hauria d&apos;utilitzar per a la majoria de comunicacions que incloguin aquest contacte?</help>
         <label>Telèfon preferit</label>
         <name>PreferredPhone__c</name>
```
Comment (skosgi1) [inline src/objectTranslations/Contact-ca.objectTranslation:3517]: @m-kinkade looks like these are removed

Comment (skosgi1) [inline src/objectTranslations/Contact-ca.objectTranslation:3517]: @m-kinkade looks like these are removed

File: src/objectTranslations/Contact-ca.objectTranslation [hunk 2]
```diff
@@ -3471,22 +3515,6 @@
...
-            <translation>Mobile Phone</translation>	
-        </picklistValues>	
-        <picklistValues>	
-            <masterLabel>Other</masterLabel>	
-            <translation>Other Phone</translation>	
-        </picklistValues>	
-        <picklistValues>	
-            <masterLabel>Work</masterLabel>	
-            <translation>Telèfon de la feina</translation>	
-        </picklistValues>
```
Comment (skosgi1) [inline src/objectTranslations/Contact-ca.objectTranslation:3489]: here too

File: src/objectTranslations/Contact-ca.objectTranslation [hunk 3]
```diff
@@ -3508,18 +3536,6 @@
...
-            <translation>Correu electrònic alternatiu</translation>	
-        </picklistValues>	
-        <picklistValues>	
-            <masterLabel>University</masterLabel>	
-            <translation>Correu electrònic universitari</translation>	
-        </picklistValues>	
-        <picklistValues>	
-            <masterLabel>Work</masterLabel>	
-            <translation>Correu electrònic de la feina</translation>	
-        </picklistValues>
```
Comment (skosgi1) [inline src/objectTranslations/Contact-ca.objectTranslation:3522]: here as well

**Comment by m-kinkade** [comment]
Replacing with https://github.com/SalesforceFoundation/EDA/pull/1232

**Pull request closed without merging**